### DISCUSSION TITLE
Value cannot be null. Parameter name: input

### DISCUSSION THREAD

**Discussion by Obegg** [zhongfly/mpv-build#89] [General]
Since I can't open a new issue I open a new discussion here.

I get the following error using the latest release and this issue doesn't happen if I use shinchiro daily releases.
![image](https://github.com/zhongfly/mpv-winbuild/assets/13978119/bf5926b3-b4d6-41fc-94a1-1ed8059ed4ce)

@zhongfly

**Comment by zhongfly** [comment]
Problem is ffmpeg.exe is broken and don't print anything in console.
Fist broken one is https://github.com/zhongfly/mpv-winbuild/actions/runs/7152161998, maybe cause by [mpv-winbuild-cmake@c55e625](https://github.com/shinchiro/mpv-winbuild-cmake/commit/c55e625ce382ed4a469b0f5530dc15aea23d6f70) 
And [clean build](https://github.com/zhongfly/mpv-winbuild/actions/runs/7230970806) make ffmpeg work again.

Download latest ffmpeg(newer than 2023.12.16 also fine) can fix.
